### DISCUSSION TITLE
Perfect Pickit of all unique and set items. Where to post it?

### DISCUSSION THREAD

**Discussion by npcoats** [blizzhackers/pickits#11] [Ideas]
HI everyone I have spent some time making a "Perfection.nip" pickit, I think I did most of everything right and would love some feedback. Also, is there a place I can post the pickit for people to download? Thank you
[Perfection.txt](https://github.com/blizzhackers/kolbot/files/6109616/Perfection.txt)

**Comment by mf022** [comment]
in https://github.com/blizzhackers/pickits/tree/master/custom was posted long time ago https://github.com/blizzhackers/pickits/blob/master/custom/perfection.nip

**Comment by mf022** [comment]
if other positive responses arrive, that file can be added in the repository.

**Comment by ckelly44** [comment]
This is awesome, thanks @npcoats! Saved me a huge headache!!!

**Comment by hexarex** [comment]
> in https://github.com/blizzhackers/pickits/tree/master/custom was posted long time ago https://github.com/blizzhackers/pickits/blob/master/custom/perfection.nip

So is OP plagiarizing? Or is it a coincidence they have the same name? I haven't compared the 2 yet._
